### ISSUE TITLE
Conv3DTranspose is not doubling the input size, instead working as Conv3D

### ISSUE THREAD

**Issue by Rajshreed** [keras-team/keras#13638]
I am using Conv3DTranspose(filters, (2,2,2), strides=(2,2,2), padding='same')(input_layer)

this halves the input layer instead of doubling it! My Keras version is 2.1.2 and tensorflow 1.4, tensorflow-gpu 1.12

**Comment by sushreebarsa** [comment]
@Rajshreed We see that you are using old version of tensorflow which is not actively supported now, please verify on 2.x and open a new issue in case you face any errors, hence moving this to closed status.Thanks!
